### PR TITLE
doc: OVHcloud account SSO connection through Azure AD

### PR DESCRIPTION
**Pull request by sebferrer**
| Question         | Answer
| ---------------- | ---
| Branch?          |  doc/user-account-connect-saml-sso-azure-ad
| Bug fix?         |  no
| New feature?     |  no
| Breaking change? |  no
| Tickets          |  UIAM-261
| License          |  BSD 3-Clause

## Description

Currently, we have written two documentations to allow users to federate their identity provider to their OVHcloud account in SAML:

- Enabling ADFS connections with your OVHcloud account: https://docs.ovh.com/us/en/customer/connect-saml-sso/

- Enabling Google Workspace SSO connections with your OVHcloud account: https://docs.ovh.com/fr/customer/connect-sso-googleworkspace/

Here's the same type of documentation to federate an Azure AD application in SAML to an OVHcloud account.

### :flags: Feature

- https://github.com/ovh/manager/pull/7527

### CODE REVIEW

File: pages/account/customer/ovhcloud-account-connect-saml-azure-ad/images/azure_ad_group_user_assignment_2.png
Comment (VincentCasse) [inline pages/account/customer/ovhcloud-account-connect-saml-azure-ad/images/azure_ad_group_user_assignment_2.png:1]: Can you hide "vincentrunabove" please?

File: pages/account/customer/ovhcloud-account-connect-saml-azure-ad/images/azure_ad_login_4.png
Comment (VincentCasse) [inline pages/account/customer/ovhcloud-account-connect-saml-azure-ad/images/azure_ad_login_4.png:1]: Can you hide vincentrunabove please?

File: pages/account/customer/ovhcloud-account-connect-saml-azure-ad/images/azure_ad_application_user_assignment_1.png
Comment (VincentCasse) [inline pages/account/customer/ovhcloud-account-connect-saml-azure-ad/images/azure_ad_application_user_assignment_1.png:1]: can you hide vincentrunabove please?

**Comment by ovh-cds** [comment]
Failed to build static files:
```sh
{{.cds.build.pelicanoutput}}
```

**Review by bs-ovh — APPROVED** [review]

**Comment by bs-ovh** [comment]
Thank you @sebferrer !

**Pull request merged**